src/en/developer-layer-example.md

```diff
@@ -1,9 +1,9 @@
 ## Writing a layer by example
 
-In this howto, we will be writing a charm layer for Vanilla. Vanilla is an open
-source themeable, pluggable, multi-lingual forum software, which enables
-communities to host a forum for discussion topics at scale. Powered by PHP and
-MySQL Vanilla is a fine example of a 3 tiered application
+In this document we will be writing a charm layer for Vanilla. Vanilla is an
+open source, themeable, pluggable, and multi-lingual forum software, which
```

Comment:
Yeah, its 2 logical thoughts mashed together with verbs and nouns and all that goodness. Maybe i'm being too picky, when read aloud it sounds fine.